### ISSUE TITLE
Sign in through steam - Noob question

### ISSUE THREAD

**Issue by theck1** [PhlexPlexico/G5V#183]
Hi.

Love what you have done!

I have this running in docker, and nginx in front. So far so good, but my question is how do I make the sign in through steam button work? 
When I click it I only get an empty page within the G5V layout. See screenshots.
I have added my API key and user account in my docker compose from Steam, but I am not sure if that it is the culprit, or I am missing something. 

Do I need another auth container towards the steam login or how is this intended to work? 
Please see attached screenshot. 

This is after I click the login button:
![image](https://github.com/user-attachments/assets/36444e41-8243-4066-b94c-e49291a7258a)

This is when I click the user icon:
![image](https://github.com/user-attachments/assets/98384cb6-3536-4525-a05e-23dae97ed130)

**Comment by theck1** [comment]
Never mind, found your nginx wiki, and got a few steps closer :)

**Comment by PhlexPlexico** [comment]
Glad you are able to get it partially working! Yes, there's a few extra steps if you're not using the Caddy docker container in the docker compose file, and using your own reverse proxy. Basically if this happens, it's due to the front-end not being able to contact the API. Once you get that linked, it should behave normally!

Thanks!

**Comment by theck1** [comment]
It sure did, Thanks! Everything seems to be working fine, however it seems anyone with a steam account can login and create matches, even though I have set only myself as admin and superadmin. 

I have also set locallogins to false in my docker compose. 

Where and how do I let only myself be able to create matches and do "admin" stuff in the web ui? 

Thanks

**Comment by PhlexPlexico** [comment]
While this is true, they would also need to provision their own servers and create their own teams. Prior to a few updates, users were unable to see their own stats unless they have logged in. 

Usually not having servers available is enough to deter most people from creating matches, as there is a lot more work involved then just logging in. And if you only need a handful of people to manage matches, you can set them as admins (or supers for rcon access during matches) and keep everything else as private.

**Comment by theck1** [comment]
So if I understand this right: 
The others that are logged in and are able to create matches can't see the servers I have added with my steam ID as admin?

**Comment by PhlexPlexico** [comment]
That is correct, and if teams attend public they won't be able to put your teams into matches either :)
